tensorflow/python/ops/image_ops_impl.py

```diff
@@ -788,6 +788,11 @@ def pad_to_bounding_box(image, offset_height, offset_width, target_height,
     If `image` was 3-D, a 3-D float Tensor of shape
     `[target_height, target_width, channels]`
 
+Usage Example:
+>>> x = tf.random.normal(shape=(256, 256, 3))
+>>> x = tf.image.pad_to_bounding_box(x, 2, 2, 260, 260)
+<x.shape = TensorShape([260, 260, 3])>
+
```

Comment:
@RichardXiao13 this failed the doctest , here is the error : 
` Failed doctest test for google3.third_party.tensorflow.python.ops.image_ops_impl.pad_to_bounding_box
 File "/google3/runfiles/google3/third_party/tensorflow/python/ops/image_ops_impl.py", line 767, in pad_to_bounding_box

----------------------------------------------------------------------
File "/runfiles/google3/third_party/tensorflow/python/ops/image_ops_impl.py", line 793, in google3.third_party.tensorflow.python.ops.image_ops_impl.pad_to_bounding_box
Failed example:
   	x = tf.image.pad_to_bounding_box(x, 2, 2, 260, 260)
Expected:
 	  <x.shape = TensorShape([260, 260, 3])>
`
can you please run the doctest locally and try?

Reply:
It seemed that I formatted the output incorrectly. It should work now!